core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -8891,4 +8891,28 @@ public Sql schema(CalciteAssert.SchemaSpec schemaSpec) {
             .withBigQuery()
             .ok(expectedBQ);
   }
+
+  @Test public void testInstrFunctionHandlingWithFourArguments() {
```

Comment:
for functions present in SqlLibraryOperators, we add test in JdbcTest and SqlFunctionsTest, and not here

Reply:
done